new-dti-website-redesign/src/components/Footer.tsx

```diff
@@ -0,0 +1,232 @@
+          width="24"
+          height="24"
+          viewBox="0 0 24 24"
+          fill="stroke-current"
+        >
+          <path
+            d="M20.8205 1.5H3.29437C2.33672 1.5 1.5 2.18906 1.5 3.13547V20.7005C1.5 21.652 2.33672 22.5 3.29437 22.5H20.8153C21.7781 22.5 22.5 21.6464 22.5 20.7005V3.13547C22.5056 2.18906 21.7781 1.5 20.8205 1.5ZM8.00953 19.0045H5.00109V9.65063H8.00953V19.0045ZM6.60938 8.22844H6.58781C5.625 8.22844 5.00156 7.51172 5.00156 6.61453C5.00156 5.70094 5.64141 5.00109 6.62578 5.00109C7.61016 5.00109 8.2125 5.69578 8.23406 6.61453C8.23359 7.51172 7.61016 8.22844 6.60938 8.22844ZM19.0045 19.0045H15.9961V13.89C15.9961 12.6647 15.5583 11.8275 14.4698 11.8275C13.6383 11.8275 13.1461 12.39 12.9272 12.938C12.8452 13.1348 12.8231 13.403 12.8231 13.6767V19.0045H9.81469V9.65063H12.8231V10.9523C13.2609 10.3289 13.9448 9.43172 15.5363 9.43172C17.5111 9.43172 19.005 10.7334 19.005 13.5398L19.0045 19.0045Z"
+            fill="currentColor"
+          />
+        </svg>
+      )
+    }
+  ];
+
+  const DTILinksSection = () => (
+    <div className="flex flex-col gap-2">
+      <h6 className="text-foreground-1">Cornell DTI</h6>
+      <ul className="flex flex-col gap-1 list-none">
+        {DTILinks.map(({ href, label }) => (
+          <li key={href}>
+            <Link
+              href={href}
+              className="text-foreground-3 hover:text-foreground-1 font-medium rounded-sm focusState transition-[color] duration-[120ms]"
+            >
+              {label}
+            </Link>
+          </li>
+        ))}
+      </ul>
+    </div>
+  );
+
+  const ProductLinksSection = () => (
+    <div className="flex flex-col gap-2">
+      <h6 className="text-foreground-1">Products</h6>
+      <ul className="flex flex-col gap-1 list-none">
+        {ProductLinks.map(({ href, label }) => (
+          <li key={label}>
+            {href ? (
+              <Link
+                href={href}
+                target="_blank"
+                className="text-foreground-3 hover:text-foreground-1 font-medium rounded-sm focusState transition-[color] duration-[120ms]"
+              >
+                {label}
+              </Link>
+            ) : (
+              <span className="text-foreground-3 cursor-default font-medium">{label}</span>
+            )}
+          </li>
+        ))}
+      </ul>
+    </div>
+  );
+
+  const DTILogoSection = () => (
+    <div>
+      <Link href={'/'} className="inline-flex rounded-sm focusState">
+        {
+          <Image
+            src="/logo.svg"
+            alt="Cornell Digital Tech & Innovation logo"
+            width={220}
+            height={50}
+          />
+        }
+      </Link>
+
+      <ul className="flex max-w-[220px] justify-between list-none pt-4">
+        {SocialLinks.map(({ href, label, icon }) => (
+          <li key={href}>
+            <Link
+              href={href}
+              className="text-foreground-3 hover:text-foreground-1 inline-flex rounded-sm focusState"
```

Comment:
```suggestion
              className="text-foreground-3 hover:text-foreground-1 inline-flex rounded-sm focusState transition-[color] duration-[120ms]"
```

just for a smoother transition! :)